samples/core/Miscellaneous/NewInEFCore6/IsNullOrWhitespaceSample.cs

```diff
@@ -87,12 +87,9 @@ public class BooksContext : DbContext
     {
         public DbSet<User> Users { get; set; }
 
-        private readonly bool _quiet;
```

Comment:
Many changes make things more explicit, but this one goes in the opposite direction and makes it implicit. Could you tell me what's the reason for this?